python/vulture/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vulture
+set _n              [string index ${name} 0]
```

Comment:
Can you explain what this bit is for? I'm not sure I get it?

Reply:
In many examples of Portfiles for PyPI packages, the single initial letter of the package name, required in `master_sites` is extracted as a substring, presumably to avoid forgetting to change it in case the name changes.
Actually `_n` is used only once, so I'm not opposed to removing the `_n` variable and writing explicitly `v` in the corresponding `master_sites`.